### ISSUE TITLE
java.lang.NoClassDefFoundError: org/apache/commons/lang/StringUtils

### ISSUE THREAD

**Issue by aheritier** [VanRoy/spring-data-jest#74]
It occurs with spring-data-jest 3.1.3 

```
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang/StringUtils
	at com.github.vanroy.springdata.jest.MappingBuilder.addSingleFieldMapping(MappingBuilder.java:258) ~[spring-data-jest-3.1.3.RELEASE.jar:?]
	at com.github.vanroy.springdata.jest.MappingBuilder.mapEntity(MappingBuilder.java:156) ~[spring-data-jest-3.1.3.RELEASE.jar:?]
	at com.github.vanroy.springdata.jest.MappingBuilder.buildMapping(MappingBuilder.java:85) ~[spring-data-jest-3.1.3.RELEASE.jar:?]
	at com.github.vanroy.springdata.jest.JestElasticsearchTemplate.putMapping(JestElasticsearchTemplate.java:229) ~[spring-data-jest-3.1.3.RELEASE.jar:?]
	at org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository.putMapping(AbstractElasticsearchRepository.java:100) ~[spring-data-elasticsearch-3.0.9.RELEASE.jar:3.0.9.RELEASE]
	at org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository.<init>(AbstractElasticsearchRepository.java:88) ~[spring-data-elasticsearch-3.0.9.RELEASE.jar:3.0.9.RELEASE]
	at org.springframework.data.elasticsearch.repository.support.SimpleElasticsearchRepository.<init>(SimpleElasticsearchRepository.java:36) ~[spring-data-elasticsearch-3.0.9.RELEASE.jar:3.0.9.RELEASE]
```

when used with  spring-data-elasticsearch 3.0.9 because `commons-lang` dep was removed:

```
[INFO] \- com.github.vanroy:spring-boot-starter-data-jest:jar:3.1.3.RELEASE:compile
[INFO]    \- com.github.vanroy:spring-data-jest:jar:3.1.3.RELEASE:compile
[INFO]       \- org.springframework.data:spring-data-elasticsearch:jar:3.0.8.RELEASE:compile
[INFO]          +- commons-lang:commons-lang:jar:2.6:compile
```
vs

```
[INFO] \- com.github.vanroy:spring-boot-starter-data-jest:jar:3.1.3.RELEASE:compile
[INFO]    \- com.github.vanroy:spring-data-jest:jar:3.1.3.RELEASE:compile
[INFO]       \- org.springframework.data:spring-data-elasticsearch:jar:3.0.9.RELEASE:compile
```

**Comment by aheritier** [comment]
Note that you should detect this kind of problem with `mvn dependency:analyze -DoutputXML=true` but it doesn't work (maybe because these are some static imports ..

**Comment by VanRoy** [comment]
@aheritier Thanks for your feedback. I fixed by removing commons-lang requirement.
It's released in 3.1.4.

**Comment by aheritier** [comment]
perfect @VanRoy it's better ! Thanks a lot
